README.md

```diff
@@ -111,6 +112,31 @@ googleAnalytics = "UA-XXXXX-X"
 
 Leave the `googleAnalytics` key empty to disable it.
 
+### Favicons
+
+Set in `Config.toml` under `[params.favicons]`
+
+We've used those generated by www.realfavicongenerator.net 
+
+If disable will fall-back to the `favicon.ico` at the root of the site
+
+ms_tile_colour sets the background colour if site is bookmarked to Windows 7/8/10 start tiles.
+
+Windows recommend that you use one of the standardised Windows colours
+* Teal 008299
+* Blue 2672EC
+* Purple 8C0095
+* Dark Purple 5133AB
+* Red AC193
+* Orange D24726
+* Green 008A00
+* Sky Blue 094AB2)
+
+Certain versions of Safari use monochrome versions of favicons, with a background colour.
+
+safari_colour sets this background colour.
+
+browserconfig.xml is for Windows Phones and Android, in the case that a bookmark is set to homescreen.
```

Comment:
I've forgotten now tbh, I'll look at the file quickly, that should make it clear to me.

manifest is android. browserconfig is windows 10/8/7/mobile.